zh_cn/README.md

```diff
@@ -0,0 +1,1627 @@
+
+# Go Concurrency Guide
+
+This guide is built on top of the some examples of the book `Go Concurrency in Go` and `Go Programming Language`
+
+[TOC]
+
+
+## 竞赛条件和数据竞赛
+
+当两个或更多的操作必须以正确的顺序执行时，就会出现竞赛条件，但程序的编写没有保证这个顺序得到维持。
+
+数据竞赛是指一个并发的操作试图读取一个变量，而在某个不确定的时间，另一个并发的操作试图写到同一个变量。主函数（main func）是主程序（goroutine）。
+
+```go
+func main() {
+    var data int
+    go func() {
+        data++
+    }()
+
+    if data == 0 {
+        fmt.Printf("the value is %d", data)
+    }
+}
+```
+
+
+## 内存访问同步化
+
+The sync package contains the concurrency primitives that are most useful for low-level memory access synchronization.
+Critical section is the place in your code that has access to a shared memory
```

Comment:
there are many things in English yet

Reply:
i forgot to delete this, i will do it later